content/plugins/provide-plugin.md

```diff
@@ -38,3 +41,11 @@ new webpack.ProvidePlugin({
   'window.jQuery': 'jquery'
 })
 ```
+
+### Use Vue.js(ES Modules)
```

Comment:
I think this feature is also effective to load ES Modules. 
(ES modules use ES2015 `export` syntax)
Please let me explain in detail with the comments below.